keel-titus-plugin/src/test/kotlin/com/netflix/spinnaker/keel/titus/resource/TitusClusterHandlerTests.kt

```diff
@@ -433,6 +440,52 @@ class TitusClusterHandlerTests : JUnit5Minutests {
             .containsDistinctElements()
         }
       }
+      context("the cluster has active server groups") {
+        before {
+          coEvery { cloudDriverService.titusActiveServerGroup("us-east-1") } returns activeServerGroupResponseEast
+          coEvery { cloudDriverService.titusActiveServerGroup("us-west-2") } returns activeServerGroupResponseWest
+        }
+
+        derivedContext<SubmittedResource<TitusClusterSpec>>("exported titus cluster spec") {
```

Comment:
I have a poor understanding of how fixtures work. Just a question, why is a derived context needed here?

Reply:
A `derivedContext` allows you to transform the fixture from the parent context into another type which sometimes makes it easier to test specific aspects of objects (like in this case the submitted resource).

Reply:
@emjburns An example I found in the docs that helped me understand: https://github.com/dmcg/minutest/blob/master/core/src/test/kotlin/dev/minutest/examples/DerivedContextExampleTests.kt

Reply:
Okay, makes sense! Thanks.